components/TextInput/TextInput.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import { action } from "@storybook/addon-actions";
+import * as React from "react";
+import TextInput, { TextInputSize } from "./TextInput";
+
+export default {
+  title: "TextInput",
+  component: TextInput
+};
+
+export const example = () => (
+  <TextInput placeholder="text input" onChange={action("inputed")} />
```

Comment:
Use `addon-knobs`'s `text`, `select`, and things like that to make all props customizable. See [`<Button>`](https://github.com/agnosticful/chipstackoverflow-web/blob/master/components/Button/Button.stories.tsx)